core/src/toga/documents.py

```diff
@@ -225,11 +225,11 @@ def __getitem__(self, path_or_index):
             return self.elements[path_or_index]
 
         # Look up by path
-        if sys.version_info < (3, 9):  # pragma: no-cover-if-gte-py39
+        if sys.version_info < (3, 10):  # pragma: no-cover-if-gte-py310
```

Comment:
This is the case that wasn't being caught by the "min + max" testing scheme. Since the problem only exists on Windows, testing 3.8 + 3.12 passed the test; 3.9 was also affected, but wasn't being checked.